src/search/components/SearchResultItem.tsx

```diff
@@ -64,14 +65,38 @@ const SearchResultItem: FunctionComponent<SearchResultItemProps> = ({
 			/>
 		);
 	};
+	const checkIfResultIsNotAnItem = (result: Avo.Search.ResultItem): boolean => {
+		return result.administrative_type === ('collectie' || 'bundel' || 'opdracht');
+	};
+
+	const renderAuthorOrOrganisation = (result: Avo.Search.ResultItem) => {
+		if (checkIfResultIsNotAnItem(result)) {
+			const name = `${result.owner?.firstname} ${result.owner?.lastname}`;
+			const initials = `${result.owner?.firstname?.[0]}${result.owner?.lastname?.[0]}`;
```

Comment:
When you use the elvis operator: ?.
you need to provide a default otherwise you'll be working with undefined
```suggestion
			const name = `${result.owner?.firstname || 'Naamloos'} ${result.owner?.lastname || ''}`;
			const initials = `${result.owner?.firstname?.[0] || ''}${result.owner?.lastname?.[0] || ''}`;
```